autoscheduler/requirements.txt

```diff
@@ -13,3 +13,4 @@ pytest-django==3.6.0
 PyPDF2==1.26.0
 requests==2.21.0
 requests-mock==1.7.0
+social-auth-app-django==3.1.0
```

Comment:
You need to add `dotenv` here, which is why the tests are failing (at least probably)

Reply:
Do I add the instructions for the .env file to the read me or just this PR?

Reply:
I think just the PR is fine